drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -379,7 +379,10 @@ local function temp_attr_handler(driver, device, ib, response)
       return
     end
     local temp = utils.round(MIRED_KELVIN_CONVERSION_CONSTANT/ib.data.value)
-    local temp_device = find_child(device, ib.endpoint_id) or device
+    local temp_device = device
+    if device:get_field(IS_PARENT_CHILD_DEVICE) == true then
+      temp_device = find_child(device, ib.endpoint_id) or device
+    end
```

Comment:
Potentially a separate commit so this is more visible outside of the blanket init fix.

Reply:
Done